docs/cookbook/express.md

```diff
@@ -1 +1,31 @@
 # Express
+
+*JSAccounts adapter for Expressjs*
+
+## Install
```

Comment:
Maybe explain before that, that we are adding in cors support inside of the package (although we want to remove it)